src/app/practice/back-line/page.tsx

```diff
@@ -1,8 +1,9 @@
-import { Exercises } from "@/components/Exercises";
+import { Exercises } from '@/components/Exercises'
```

Comment:
Since we re-organized to use `/conditioning` & `/restorative` instead of /practice, you can remove the entire practice directory with the command `rm -rf practice`